sdk/containerregistry/azure-containerregistry/azure/containerregistry/aio/_async_container_repository_client.py

```diff
@@ -53,25 +53,25 @@ async def delete(self, **kwargs) -> None:
         """
         await self._client.container_registry.delete_repository(self.repository, **kwargs)
 
-    def delete_registry_artifact(self, digest: str, **kwargs) -> None:
+    async def delete_registry_artifact(self, digest: str, **kwargs) -> None:
         """Delete a registry artifact
 
         :param digest: The digest of the artifact to be deleted
         :type digest: str
         :returns: None
         :raises: :class:~azure.core.exceptions.ResourceNotFoundError
         """
-        raise NotImplementedError("Has not been implemented")
+        await self._client.container_registry_repository.delete_manifest(self.repository, digest, **kwargs)
 
-    def delete_tag(self, tag: str, **kwargs) -> None:
+    async def delete_tag(self, tag: str, **kwargs) -> None:
```

Comment:
haven't looked through all your PRs yet so might have missed it, but do you plan to add the distributed tracing decorators to your client methods?

Reply:
Added!